UefiCpuPkg/PiSmmCpuDxeSmm/PiSmmCpuDxeSmm.c

```diff
@@ -1153,7 +1153,7 @@ PiCpuSmmEntry (
   Status                 = GetSmBase (mMaxNumberOfCpus, &mCpuHotPlugData.SmBase);
   if (Status == EFI_OUT_OF_RESOURCES) {
     ASSERT (Status != EFI_OUT_OF_RESOURCES);

```

Comment:
Line 707 of this file also has a deadloop. I assume you would like to change that too?

Reply:
Updated.